packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -73,6 +73,8 @@ const trace = makeTracer('VM');
 /**
  * @typedef {object} MetricsNotification
  *
+ * @property {Ratio | undefined} lockedQuote      priceQuote that will be used for liquidation.
+ *                                   Non-null from priceLock time until liquidation has taken place.
```

Comment:
technically "undefined" is not null.

To be explicit about "no locked quote" vs indeterminate or missing key let's use `null` instead of `undefined`.

Reply:
done.